css/selectors/checked.json

```
@@ -7,34 +7,41 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:checked",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "chrome_android": {
               "version_added": "18"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "12",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Support for styling <code>&lt;option&rt;</code> elements was removed in Firefox 56.  Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "firefox_android": {
-              "version_added": "4"
+              "version_added": "4",
+              "notes": "Support for styling <code>&lt;option&rt;</code> elements was removed in Firefox 56."
             },
             "ie": {
               "version_added": "9"
             },
             "opera": {
-              "version_added": "9"
+              "version_added": "9",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
             },
             "opera_android": {
               "version_added": "10.1"
             },
             "safari": {
-              "version_added": "3.1"
+              "version_added": "3.1",
+              "notes": "Due to OS limitations, styling of <code>&lt;option&rt;</code> elements does not work on macOS."
```

Desktop Safari is not supported on other platforms, so we can omit the discussion of the operating system entirely

```suggestion
              "notes": "Styling <code>&lt;option&rt;</code> elements has no effect."
```